generators/app/templates/ext-jupyterbook/src/jupyter-book.ts

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+import * as vscode from 'vscode';
+import * as azdata from 'azdata';
+import * as path from 'path';
+import * as fs from 'fs';
+import * as os from 'os';
+
+const processNotebooks = () => {
+    const rootExtensionsFolder = path.normalize(path.join(os.homedir(), '.azuredatastudio', 'extensions'))
```

Comment:
please add comment for each section explaining what it those. if this is what user gets after generating, it would be nice to add comment to help user change the code for their need